stock_move_description_according_sale_order_line/models/stock_move.py

```diff
@@ -0,0 +1,18 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (http://www.gnu.org/licenses/lgpl-3.0.html)
+from odoo import models, api
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        ret = super().create(vals_list)
+        for move in ret:
```

Comment:
Make changes only for those moves that come from a sale line:
```suggestion
        for move in ret.filtered(lambda x: x.sale_line_id):
```